pkg/controller/migmigration/task.go

```diff
@@ -163,8 +162,10 @@ var FinalItinerary = Itinerary{
 		{Name: CleanStaleStagePods, Step: StepPrepare},
 		{Name: WaitForStaleStagePodsTerminated, Step: StepPrepare},
 		{Name: CreateRegistries, Step: StepPrepare},
-		{Name: EnsureCloudSecretPropagated, Step: StepPrepare},
+		{Name: WaitForVeleroReady, Step: StepPrepare},
+		{Name: WaitForResticReady, Step: StepPrepare, any: HasPVs},
 		{Name: WaitForRegistriesReady, Step: StepPrepare},
+		{Name: EnsureCloudSecretPropagated, Step: StepPrepare},
```

Comment:
I had to move this phase so that we wait for Velero to be ready before execing into the Pod.